<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1812</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quad boolean</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QFrame" name="toolsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="loadMeshesPushButton">
         <property name="text">
          <string>Load Meshes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="moveCenterCheckBox">
         <property name="text">
          <string>Move center</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="computeAllPushButton">
         <property name="text">
          <string>Compute all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quadTracerPushButton">
         <property name="text">
          <string>Quad tracer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="computeBooleanPushButton">
         <property name="text">
          <string>Compute boolean</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="smoothPushButton">
         <property name="text">
          <string>Smooth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getSurfacesPushButton">
         <property name="text">
          <string>Get surfaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decompositionPushButton">
         <property name="text">
          <string>Decomposition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ilpPushButton">
         <property name="text">
          <string>ILP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quadrangulatePushButton">
         <property name="text">
          <string>Quadrangulate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getResultPushButton">
         <property name="text">
          <string>Get result</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveResultPushButton">
         <property name="text">
          <string>Save result</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="settingsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="motorcycleCheckBox">
         <property name="text">
          <string>Motorcycle</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="operationFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="operationUnionRadio">
            <property name="text">
             <string>U</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="operationIntersectionRadio">
            <property name="text">
             <string>I</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="operationDifferenceRadio">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="intersectionSmoothingLabel">
         <property name="text">
          <string>Int sm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intersectionSmoothingSpinBox">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="intersectionSmoothingNRingLabel">
         <property name="text">
          <string>N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intersectionSmoothingNRingSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="minRectangleSideLabel">
         <property name="text">
          <string>Min rectangles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minRectangleSideSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mergeCheckBox">
         <property name="text">
          <string>Merge</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="deleteSmallCheckBox">
         <property name="text">
          <string>Delete small</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="deleteNonConnectedCheckBox">
         <property name="text">
          <string>Delete non-connected</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="alphaLabel">
         <property name="text">
          <string>Alpha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="alphaSpinBox">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>0.990000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="initialRemeshingCheckBox">
         <property name="text">
          <string>Remeshing</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="edgeFactorLabe">
         <property name="text">
          <string>E fact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="edgeFactorSpinBox">
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="chartSmoothingLabel">
         <property name="text">
          <string>Chart sm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="chartSmoothingSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="meshSmoothingLabel">
         <property name="text">
          <string>Quad sm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="meshSmoothingSpinBox">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultSmoothingLabel">
         <property name="text">
          <string>Result sm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="resultSmoothingSpinBox">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultSmoothingNRingLabel">
         <property name="text">
          <string>N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="resultSmoothingNRingSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="actionFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="resetTrackballButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="trackSceneButton">
         <property name="text">
          <string>Track scene</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="track1Button">
         <property name="text">
          <string>Track 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="track2Button">
         <property name="text">
          <string>Track 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showWireframe">
         <property name="text">
          <string>Wireframes</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showMesh1CheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Mesh 1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showMesh2CheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Mesh 2</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showQuadLayout1CheckBox">
         <property name="text">
          <string>L1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showQuadLayout2CheckBox">
         <property name="text">
          <string>L2</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showBooleanCheckBox">
         <property name="text">
          <string>Boolean</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showIntersectionCurvesCheckBox">
         <property name="text">
          <string>Intersection curves</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showPreservedSurfaceCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Preserved</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showQuadLayoutPreserved1CheckBox">
         <property name="text">
          <string>L1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showQuadLayoutPreserved2CheckBox">
         <property name="text">
          <string>L2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showNewSurfaceCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showChartSidesCheckBox">
         <property name="text">
          <string>Sides</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showILPCheckBox">
         <property name="text">
          <string>ILP </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showQuadrangulatedCheckBox">
         <property name="text">
          <string>Quadrangulated</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showQuadrangulatedLayoutCheckBox">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showResultCheckBox">
         <property name="text">
          <string>Result</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="glFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="GLArea" name="glArea" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1812</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLArea</class>
   <extends>QWidget</extends>
   <header>glarea.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>loadMeshesPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
